build.gradle

```diff
@@ -117,6 +117,14 @@ dependencies {
         exclude group: 'org.mortbay.jetty', module: 'servlet-api'
     }
     testCompile "com.ofg:micro-infra-spring-test:$microInfraSpringVersion"
+
+    testCompile "org.gebish:geb-core:0.10.0"
+    testCompile "org.gebish:geb-spock:0.10.0"
+    testCompile "org.spockframework:spock-spring"
+    testCompile "org.seleniumhq.selenium:selenium-firefox-driver:2.43.1"
+    testCompile "org.seleniumhq.selenium:selenium-support:2.43.1"
+    testCompile "org.seleniumhq.selenium:selenium-htmlunit-driver:2.43.1"

```

Comment:
I would move repeatable versions to gradle.properties (like `microInfraSpringVersion` already).